api/cases/case.js

```diff
@@ -48,16 +59,18 @@ module.exports = (req, res, next) => {
     const token = req.auth.token;
     const userId = req.auth.userId;
     const caseId = req.params.case_id;
-    
-    getCase(caseId, userId, {
+
+    getCaseWithEvents(caseId, userId, {
         headers : {
             'Authorization' : `Bearer ${token}`,
             'ServiceAuthorization' : req.headers.ServiceAuthorization
         }
-    }).then(caseData => {
+    }).then( responses => {
+        const caseData = responses[0];
+        caseData.events = responses[1];
+
```

Comment:
you could use destructoring for the params ie. `.then({allData, eventsData} =>...`